parent/pom.xml

```diff
@@ -107,7 +107,7 @@
         <jsonPath.version>2.0.0</jsonPath.version>
         <commons-compress.version>1.4</commons-compress.version>
         <qpid.version>0.20</qpid.version>
-        <mongodb.version>2.11.4</mongodb.version>
+        <mongodb.version>3.0.3</mongodb.version>
```

Comment:
The 3.x client is compatible with 2.x[1], and addresses intermittent connection issues that we were seeing with the 2.11.4 client

[1]: http://docs.mongodb.org/ecosystem/drivers/driver-compatibility-reference/#java-driver-compatibility